pom.xml

```diff
@@ -85,6 +85,7 @@
         <version.org.testng>6.14.2</version.org.testng>
         <version.resteasy.testsuite>${version.org.jboss.resteasy}</version.resteasy.testsuite>
         <version.junit>4.13.2</version.junit>
+        <server.version>23.0.2.Final</server.version>
```

Comment:
This could be 24.0.0.Final. 24.0.1.Final should be released this week though.

Reply:
will change it.